mysql-test/include/log_encrypt_6.inc

```diff
@@ -85,7 +85,7 @@ DROP DATABASE tde_db;
 
 --echo # Try starting without keyring : Error
 let NEW_CMD = $MYSQLD --no-defaults --innodb_dedicated_server=OFF --innodb_page_size=$START_PAGE_SIZE --innodb_log_file_size=$LOG_FILE_SIZE --basedir=$MYSQLD_BASEDIR --datadir=$MYSQLD_DATADIR1  --secure-file-priv="" --console </dev/null>>$MYSQL_TMP_DIR/wl9290.err 2>&1;
---error 1,42
```

Comment:
Likewise

Reply:
Double checked with `mtr --debug-server --force log_encrypt_6_rk log_encrypt_6_mk --sanitize`.
42 here is needed to suppress the same `sync_latch_meta_init()` memory leak.